### DISCUSSION TITLE
Can't install GitHub CLI Arch Linux

### DISCUSSION THREAD

**Discussion by ghost** [community/community#39253] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hey guys.

There is a problem with GitHub CLI when I'm trying to install it in Arch Linux. I provide you with a screenshot so you can see the message the terminal displays.

I'd appreciate the help.

![1](https://user-images.githubusercontent.com/91437129/202215503-afd5cea1-1439-4974-9251-676bd935f6f0.png)

**Comment by DSDmark** [comment, marked as answer]
Hi @imjavierpalma,

did you upgrade your system first?. I think maybe cause this issue.

Try this ```sudo pacman -Syu github-cli```.
If it's not work, then I think you can use **Snap** to install that.
```sudo snap install gh```

**Reply by ghost** [reply]
It worked! I forgot the update my system before installing GitHub CLI.

Thanks for the help! @DSDmark
